docs/acme-divergences.md

```diff
@@ -6,6 +6,10 @@ This document details these differences, since ACME is not yet finalized it will
 
 Current draft: [`draft-ietf-acme-acme-04`](https://tools.ietf.org/html/draft-ietf-acme-acme-04).
 
+## [Section 5](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-5)
+
+Boulder does not implement the [general JWS syntax](https://tools.ietf.org/html/rfc7515#page-20), but only accepts the [flattened syntax](https://tools.ietf.org/html/rfc7515#page-21). All examples in draft-ietf-acme-04 are using the supported flattened syntax.
```

Comment:
>  All examples in draft-ietf-acme-04 are using the supported flattened syntax.

Can you remove this last sentence? I don't think its necessary - the RFC already mentions that the examples use the flattened syntax.